README.md

```diff
@@ -1 +1,256 @@
+
+If you think that change is necessary - open a Pull Request in this repository and let your code reviewer know about it.
+4. Run npx stylelint "**/*.{css,scss}" on the root of your directory of your project.
+
+5. Fix linter errors.
+
+6. **IMPORTANT NOTE:** feel free to research auto-correct options for Stylelint if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+
+### Making Your HTML and CSS Code From Figma
+-Pick a figma template to work on and use css and html to build a replica the first 2 sections of the mobile website using the template you have chosen.
+
+-Make sure to replace the contents of helloworld.html with new contents of your own html code which replicates the chosen figma template. 
+
+-In your figma template, you will create the toolbar (or header) and the headline section (right after the header)
+
+**note**
+-Make sure to pick the mobile template on figma and not the desktop template.
+-In order to lay out the elements on the page you should use Flexbox in all 2 sections.
+-You don't need to implement any functionality that requires JavaScript, like opening the menu or creating dropdowns.
+-You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using the templates in Figma.
+-Implement the button interactions (enable, hover, etc.).
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Joy Phoebe**
+
+- GitHub: (https://github.com/joyapisi)
+- Twitter: (https://twitter.com/joyapisi)
+- LinkedIn: (https://http://www.linkedin.com/in/joy-phoebe-00b80a13a)
+
+
+## 🤝 Contributing <a name="contributing"></a>
+ Benson Njuguna: (https://github.com/Bennyjoez)
+  Fatiha Jeddaa: (https://github.com/Jeddaa)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+# **[Add_popup_button]**
+In my speakers section, I would like to make the view more button reveal more speeakers using DOM manipulation
+# **[Create_other_pages]**
+This website only has the home page and about page. In the future, I would like to add other pages to it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/joyapisi/ThePortfolio/issues).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly leave a comment below and share it with someone who enjoys coding! Coding is all about continuous learning and allowing yourself to be a beginner. Keep going! 
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="Microverse Inc."></a>
+
+I'm thankful to Microverse for providing a study platform which guided me through this project and to my coding partners at Microverse for the collaborative effort. 
```

Comment:
- [ ] Kindly make sure you acknowledged the owner of the original design in your README as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.) 👍